src/Tizen.NUI/src/public/BaseComponents/VectorGraphcis/CanvasView.cs

```diff
@@ -0,0 +1,109 @@
+/*
+* Copyright(c) 2019 Samsung Electronics Co., Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+using System;
+using System.ComponentModel;
+
+
+namespace Tizen.NUI.BaseComponents
+{
+    namespace VectorGraphics
+    {
+        /// <summary>
+        /// CanvasView is a class for displaying vector primitives.<br />
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public class CanvasView : View
+        {
+            
+            static CanvasView() { }
+
+            /// <summary>
+            /// Creates an initialized CanvasView.
+            /// </summary>
+            /// <param name="viewBox">The size of viewbox.</param>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public CanvasView(Size2D viewBox) : this(Interop.CanvasView.New(Uint16Pair.getCPtr( new Uint16Pair((uint)viewBox.Width, (uint)viewBox.Height))), true)
```

Comment:
Please check build warnings: the constructor is public so the given `viewBox` can be null. 
In c#, it is normal to throw an `ArgumentNullException` when the given parameter is null but it is not allowed.
So in this case, this can be a one of solution.
```C#
public CanvasView(Size2D viewBox) : this(viewBox == null ? throw new ArgumentNullException(nameof(viewBox)) : Interop.CanvasView.New(Uint16Pair.getCPtr( new Uint16Pair((uint)viewBox.Width, (uint)viewBox.Height))), true)
```

Reply:
Added throw code. thanks